quickstart.md

```diff
@@ -164,250 +166,51 @@ enclave-cc      enclave-cc      9m55s
 
 While the operator deploys all the required binaries and artifacts and sets up runtime classes that use them,
-certain platforms may require additional configuration to enable confidential computing. For example, the host
-kernel and firmware might need to be configured.
-See the [guides](./guides) for more information.
+certain platforms may require additional configuration to enable confidential computing. For example, a specific 
+host kernel or firmware may need to be utilized. The guides section has further instructions on how to apply
+individual TEE environments.
 
-# Running a workload
+## Using CoCo
 
-## Creating a sample CoCo workload
+The CoCo operator environment has been deployed and setup!
 
-Once you've used the operator to install Confidential Containers, you can run a pod with CoCo by simply adding a runtime class.
-First, we will use the `kata` runtime class which uses CoCo without hardware support.
-Initially we will try this with an unencrypted container image.
+Below is a brief summary and description of some of the CoCo use cases:
 
-In our example we will be using the bitnami/nginx image as described in the following yaml:
-```
-apiVersion: v1
-kind: Pod
-metadata:
-  labels:
-    run: nginx
-  name: nginx
-  annotations:
-    io.containerd.cri.runtime-handler: kata
-spec:
-  containers:
-  - image: bitnami/nginx:1.22.0
-    name: nginx
-  dnsPolicy: ClusterFirst
-  runtimeClassName: kata
-```
+- **Container Launch with Only Memory Encryption (No Attestation)** - Launch a container with memory encryption 
+- **Container Launch with Encrypted Image** - Launch an encrypted container by proving the workload is running 
+  in a TEE in order to retrieve the decryption key
+- **Container Launch with Image Signature Verification** - Launch a container and verify the authenticity and 
+  integrity of an image by proving the workload is running in a TEE
+- **Sealed secret** - Implement wrapped kubernetes secrets that are confidential to the workload owner and are
+  automatically decrypted by proving the workload is running in a TEE
 
-Setting the runtimeClassName is usually the only change needed to the pod yaml, but some platforms
-support additional annotations for configuring the enclave. See the [guides](./guides) for
-more details.
+The CoCo use cases are implemented differently for each TEE, and they are described in each corresponding 
+[guide](./guides) section. To get started using CoCo without TEE hardware, follow the Non-TEE guide below:
 
-With Confidential Containers, the workload container images are never downloaded on the host.
-For verifying that the container image doesn’t exist on the host you should log into the k8s node and ensure the following command returns an empty result:
-```
-root@cluster01-master-0:/home/ubuntu# crictl  -r  unix:///run/containerd/containerd.sock image ls | grep bitnami/nginx
-```
-You will run this command again after the container has started.
+- [Non-TEE](./guides/non-tee.md)
```

Comment:
The way this PR split out the original quickstart material, non-tee.md looks like it contains TEE content.